modernizer-maven-plugin/src/test/java/org/gaul/modernizer_maven_plugin/ModernizerTestHelper.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gaul.modernizer_maven_plugin;
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class ModernizerTestHelper {
```

Comment:
Can you call this `ModernizerSuppressionsEndToEndTestClass`? `ModernizerTestHelper` makes it seem as though it provides helper methods, when really it's the input.